app/controllers/ad_privacy_policy_controller.rb

```diff
@@ -4,6 +4,19 @@ class AdPrivacyPolicyController < ApplicationController
   before_action :authenticate_user!
 
   def show
-    @reg_identifier = params[:reg_identifier]
+    @continue_link = continue_link
+  end
+
+  private
+
+  def continue_link
+    reg_identifier = params[:reg_identifier]
+    renewing_registration = WasteCarriersEngine::RenewingRegistration.where(reg_identifier: reg_identifier).first
+
+    if renewing_registration.present?
+      WasteCarriersEngine::Engine.routes.url_helpers.new_renewal_start_form_path(token: renewing_registration.token)
+    else
+      WasteCarriersEngine::Engine.routes.url_helpers.new_renewal_start_form_path(token: reg_identifier)
```

Comment:
We have code in place that will redirect to the correct journey page for renewals whenever you hit the `renew` start page with a reg_identifier, so we don't need this at all :) just always send the `token` to the new renewal start form path as a `token`

Reply:
That was what I tried at first but it didn't work! I'll try again and see if I overlooked something obvious.

Reply:
Oh! We have merged that fix yesterday I think. If it doesn't work anyway, let me know as that should work, so I'll fix it :)

Reply:
Ahhhhhhhhhh. That explains it. Will update the engine gem!

Reply:
Yes, that solved it. Will rework the PR.

Reply:
Cool stuff :)